packages/shopify-cart/src/client/actions/cartCreate.ts

```diff
@@ -24,12 +31,19 @@ export default async function cartCreate({
   params
 }: CreateCartParams): Promise<void | CartResponse> {
   try {
+    let formattedParams: NacelleCartInput | CartInput | undefined = params;
+    if (params && params.lines) {
+      const { lines: nacelleLines = [] } = params;
```

Comment:
If we make this change it boosts code coverage a bit without needing a new test

```suggestion
      const nacelleLines = params.lines;
```